connectors/src/connectors/intercom/lib/help_center_permissions.ts

```diff
@@ -0,0 +1,356 @@
+import type { Client as IntercomClient } from "intercom-client";
+import { Op } from "sequelize";
+
+import {
+  fetchIntercomArticle,
+  fetchIntercomArticles,
+  fetchIntercomCollection,
+  fetchIntercomCollections,
+  fetchIntercomHelpCenter,
+} from "@connectors/connectors/intercom/lib/intercom_api";
+import type { Connector } from "@connectors/lib/models";
+import {
+  IntercomArticle,
+  IntercomCollection,
+  IntercomHelpCenter,
+} from "@connectors/lib/models/intercom";
+import logger from "@connectors/logger/logger";
+
+// A Help Center contains collections and articles:
+// - Level 1: Collections (parent_id is null)
+// - Level 2: Collections (parent_id is a level 1 collection)
+// - Level 3: Collections (parent_id is a level 2 collection)
```

Comment:
Is there at most 3 levels, so `collection -> collection -> article` is the most nested structure ?

Reply:
Or is it `collection -> collection -> collection -> article` ?

Reply:
thanks